rotkehlchen/exchanges/coinbase.py

```diff
@@ -344,22 +444,15 @@ def _api_query(
             if 'data' not in json_ret:
                 raise RemoteError(f'Coinbase json response does not contain data: {response.text}')
 
-            # `data` attr is a list in itself
```

Comment:
Here and in all the other lines why are you removing all these comments? What is the purpose of this change?